Makefile

```diff
@@ -25,6 +25,26 @@ export DEFAULT_ROUTING ?= basic
 export KUBECONFIG ?= ${HOME}/.kube/config
 export DEVWORKSPACE_API_VERSION ?= 1f335562c475972132851c68227dd36558317bb3
 
+# By default the project should be build under GOPATH/src/github.com/<orgname>/<reponame>
```

Comment:
That's not true for `go mod`
we should hardcode it or read from `https://github.com/devfile/devworkspace-operator/blob/main/go.mod#L1`

Reply:
Done, reading from go mod instead!